server/http/helloapp/f_send_survey.go

```diff
@@ -0,0 +1,89 @@
+package helloapp
+
+import (
+	"net/http"
+	"strings"
+
+	"github.com/mattermost/mattermost-server/v5/model"
+
+	"github.com/mattermost/mattermost-plugin-apps/server/apps"
+	"github.com/mattermost/mattermost-plugin-apps/server/utils/httputils"
+)
+
+func (h *helloapp) newSendSurveyFormResponse(claims *apps.JWTClaims, c *apps.Call) *apps.CallResponse {
+	return &apps.CallResponse{
+		Type: apps.CallResponseTypeForm,
+		Form: &apps.Form{
+			Title:  "Send a survey to user",
+			Header: "Message modal form header",
+			Footer: "Message modal form footer",
+			Fields: []*apps.Field{
+				{
+					Name:                 fieldUserID,
+					Type:                 apps.FieldTypeUser,
+					Description:          "User to send the survey to",
+					Label:                "User",
+					AutocompleteHint:     "enter user ID or @user",
+					AutocompletePosition: 1,
+					ModalLabel:           "User",
+				}, {
+					Name:             fieldMessage,
+					Type:             apps.FieldTypeText,
+					IsRequired:       true,
+					Description:      "Text to ask the user about",
+					Label:            "message",
+					AutocompleteHint: "Anything you want to say",
+					ModalLabel:       "Text",
+					TextMinLength:    2,
+					TextMaxLength:    1024,
+				},
+			},
```

Comment:
`Field.Value` is currently how we set the default value of a field. Are we planning on having that property be the source of truth value on a refresh? Or are we going to have a separate `map[string]string` sent to the client for the form refresh?

Reply:
I was thinking, at least for now - the former (ack that we may need to extend it)